packages/checkout-ui-extensions/src/extension-points/api/standard/index.ts

```diff
@@ -600,10 +608,15 @@ export interface StandardApi<
   /**
    * The settings values matching the settings schema defined in the `shopify.ui.extension.toml` file.
    *
-   * NOTE: When an extension is being installed in the editor, the settings values will be empty until
+   * **Note:** When an extension is being installed in the editor, the settings values will be empty until
    * a merchant sets a value. In that case, this object will be updated in real time as a merchant fills in the settings.
    */
   settings: StatefulRemoteSubscribable<ExtensionSettings>;
+
+  /**
+   * Exposes a `analytics.publish` method to publish analytics events.
+   */
+  analytics: Analytics;
```

Comment:
I think that line and the related types are the only changes that are not just comments changes.